prowler/lib/utils/gc.py

```diff
@@ -0,0 +1,35 @@
+import gc
+
+from contextlib import contextmanager
+
+
+@contextmanager
+def force_gc(disable_gc=False):
```

Comment:
What is the point of this context manager? Is it intended for testing and benchmarking?